website/src/hugo/content/versions.md

```diff
@@ -206,7 +206,7 @@ title: Versions
       </table>
     </div>
     <div role="tabpanel" class="tab-pane" id="changelog">
-      {{< readfile "CHANGELOG.md" >}} 
+      {{< readfile "content/CHANGELOG.md" >}} 
```

Comment:
This buries the changelog pretty deep in the directory structure, but avoids a hacky copy in the build.  Hugo can't read outside its parent directory, probably because security and stuff.

An alternative would be a symlink, but I think that's still shady on Windows, and we have a few such developers.